src/Parser/Warrior/Arms/CombatLogParser.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import SuggestionsTab from 'Main/SuggestionsTab';
+import Tab from 'Main/Tab';
+import Talents from 'Main/Talents';
+
+import CoreCombatLogParser from 'Parser/Core/CombatLogParser';
+import DamageDone from 'Parser/Core/Modules/DamageDone';
+
+import CastEfficiency from './Modules/Features/CastEfficiency';
+import AlwaysBeCasting from './Modules/Features/AlwaysBeCasting';
+import CooldownTracker from './Modules/Features/CooldownTracker';
+import ColossusSmashUptime from './Modules/BuffDebuff/ColossusSmashUptime';
+
+//import RelicTraits from './Modules/Traits/RelicTraits';
```

Comment:
Any reason Relic traits are disabled?

Reply:
Due to long research and analyzing about which is usefull to show, I'm going to work on it in the next PR if you allow it.

Reply:
That is fine. Just was asking is all.